### PR TITLE
Fix documentation rendering

### PR DESCRIPTION
**Pull request by jocgir**
- Some contents forced the HTML help to scroll.
- Generate separated code block for each example
- Add some more examples

### CODE REVIEW

**Comment by julienduchesne** [comment]
Wow nice, I really like the new layout

File: render-doc [hunk 1]
```diff
@@ -4,14 +4,32 @@ go build -o gotemplate
...
 mkdir -p $DOC_FOLDER
 
+git submodule update --init
+
+echo '
+.flink {
+    display: inline-block;
+    min-width: 2.5in;
+}
+' > docs/themes/book/assets/_custom.scss
```
Comment (julienduchesne) [inline render-doc]: You don't have to put that in the book theme directly. You create an `assets` folder beside content. We don't need to generate this dynamically. Just commit the scss file

Comment (julienduchesne) [reply render-doc]: https://github.com/alex-shpak/hugo-book#extra-customisation

Comment (jocgir) [reply render-doc]: OK, thanks.

**Pull request merged**